### ISSUE TITLE
[Task] Interactive command - help: Display command list

### ISSUE THREAD

**Issue by taorepoara** [lenra-io/lenra_cli#28]
## What should be done 
Implement the `help` interactive command.

Displays the interactive commands list and explains how to use them.

## Technical recommandation
Use the `clap` crate help management.

## Is this task linked with any other ?
Blocked by https://github.com/lenra-io/lenra_cli/issues/21

**Comment by taorepoara** [comment]
By default in clap, so already done.
